scripts/watch.js

```diff
@@ -2,20 +2,27 @@
 const DEVELOPMENT = 'development';
 process.env.NODE_ENV = DEVELOPMENT;
 
-const fs = require('fs-extra');
-const paths = require('../config/paths');
+//const fs = require('fs-extra');
+//const paths = require('../config/paths');
 const webpack = require('webpack');
 const config = require('../config/webpack.config.js');
 const devConfig = config(DEVELOPMENT);
 
-webpack(devConfig).watch({}, (err, stats) => {
-	if (err) {
-		console.error(err);
+webpack(devConfig).watch(
+	{
+		poll: 500,
+		aggregateTimeout: 500,
+		ignored: /node_modules/,
+	},
```

Comment:
polling just stopped for me suddenly (around the same time I had all of that other wonky stuff happen - looking at you MS!)
so had to set the polling params here to get it working